src/components/Title/index.js

```diff
@@ -13,7 +13,9 @@ const Title = ({title, subTitle, smaller, isFullScreen, className}) =>
       className
```

Comment:
Minor: I think it might make sense to rename this component `PageTitle`, to make it quite clear that it should be used only as a title of a page. thoughts?

Reply:
Yes, that would make it more semantic but I would leave it outside of this PR as it's blocking other accessibility work

Reply:
cool, is it okay to follow up on another PR on the same issue? that way we unblock, but we can still address some points.

Reply:
Sounds good. I can open a new PR to change the component name

Reply:
https://github.com/onfido/onfido-sdk-ui/pull/639